fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -55,6 +55,11 @@ def fetch_acs_income(
         data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
         DataFrames or Series as describe above.
 
+        .. note::
+            Default value changed to True.
+
+        .. versionchanged:: 0.9.0
```

Comment:
nit: you can actually combine the note and versionchanged!
See https://github.com/fairlearn/fairlearn/blob/c09c2ea4a1f6c4c25d0941212b59cf7475f4e518/fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py for example. The note just goes right under versionchanged. I believe they end up in the same box that way. OK, done nitpicking now 😆

Reply:
Oooh that's actually really nice. 🤓 Never stop nitpicking as long as there's room for improvement, haha.